### ISSUE TITLE
JSON Schema Draft 8 implementation

### ISSUE THREAD

**Issue by halex2005** [fastify/fluent-schema#49]
Hi!
There is new JSON Schema draft available here: http://json-schema.org/specification-links.html#2019-09-formerly-known-as-draft-8
Does this library needs changes?
Actually, I don't read this draft yet. May be someone tell: does this library already implements new draft or it needs some changes?

I suggest you create roadmap for the changes, if necessary.
It could help the community to focus what's really needed and track the overall progress.

**Comment by Eomm** [comment]
Thanks for the advice

There are a couple of things that are really breaking changes like:

-  "definitions" renamed to "$defs"
- "dependencies" renamed to "dependentRequired"

Plus new keywords

Help is always appreciate 💪🏻

// cc @mcollina per fast-json-stringify

**Comment by aboutlo** [comment]
good point @halex2005 IMO until we don't have a validator eg. AJV there isn't too much value. 
On the other hand in order to support draft 7 and draft 8 we need to rework the implementation and polishing the tests to make clear what we support on each version. 

They are talking here as well: https://github.com/epoberezkin/ajv/issues/1082

**Comment by aboutlo** [comment]
A more updated issue on AJV side: https://github.com/ajv-validator/ajv/issues/1198

**Comment by mcollina** [comment]
fast-json-stringify does not implement the latest draft.

**Comment by onosendi** [comment]
> fast-json-stringify does not implement the latest draft.

When will it implement the latest draft?

**Comment by mcollina** [comment]
> When will it implement the latest draft?

When somebody will PR that in
